appinfo/Migrations/Version20190315192717.php

```diff
@@ -252,9 +268,50 @@ private function getTranslatedLegacyConfig($prefix) {
 			}
 		}
 
+		foreach ($this->arrayConfigKeys as $arrayConfigKey) {
+			if (isset($current[$arrayConfigKey])) {
+				$current[$arrayConfigKey] = $this->convertToArray($current[$arrayConfigKey]);
+			}
+		}
+
 		return $current;
 	}
 
+	/**
+	 * @param  mixed $value
+	 * @return string[]
+	 */
+	private function convertToArray($value) {
+		if (empty($value)) {
```

Comment:
maybe include some comments to make it easier to follow?

Reply:
done